.github/workflows/ci.yml

```diff
@@ -86,38 +93,50 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install gcc-multilib
-      - name: Test almost no features
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          target: ${{ matrix.target }}
+          override: true
+      - name: Cache Builds
+        uses: Swatinem/rust-cache@v1
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Test (Minimal)
         uses: actions-rs/cargo@v1
-        if: matrix.features == 'none'
+        if: matrix.features == 'minimal'
         with:
           command: test
-          args: --target ${{ matrix.target }} --no-default-features --features "std cargo" -p clap:3.0.0-beta.5
-      - name: Test all features
+          args: --target ${{ matrix.target }} ${{ needs.args.outputs.minimal }}
+      - name: Test (Full)
         uses: actions-rs/cargo@v1
-        if: matrix.features == 'all'
+        if: matrix.features == 'full'
         with:
           command: test
-          args: --target ${{ matrix.target }} --features "wrap_help yaml regex unstable-replace unstable-multicall unstable-grouped"
+          args: --target ${{ matrix.target }} ${{ needs.args.outputs.full }}
       - name: Check debug
         uses: actions-rs/cargo@v1
-        if: matrix.features == 'all'
+        if: matrix.features == 'full'
         with:
           command: check
-          args: --target ${{ matrix.target }} --features "wrap_help yaml regex unstable-replace unstable-multicall unstable-grouped debug"
-      - name: Test release
+          args: --target ${{ matrix.target }} ${{ needs.args.outputs.full }} --features debug
```

Comment:
If we are wanting to document for users, shouldn't we be getting the debug flag from the Makefile as well?